app/controllers/admin/draft_edition_change_notes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Admin::DraftEditionChangeNotesController < Admin::BaseController
+  before_action :find_edition, :enforce_permissions!, :redirect_to_show_page_unless_editable
+  layout "design_system"
+
+  def edit
+    @change_note_form = ChangeNoteForm.build_from_edition(@edition)
+  end
+
+  def update
+    @change_note_form = ChangeNoteForm.new(change_note_params)
+
+    if @change_note_form.save!(@edition)
+      redirect_to admin_edition_path(@edition), notice: "Change note updated successfully"
+    else
+      render :edit
+    end
+  end
+
+private
+
+  def find_edition
+    @edition = Edition.find(params[:edition_id])
+  end
+
+  def enforce_permissions!
+    enforce_permission!(:update, @edition)
+  end
+
+  def redirect_to_show_page_unless_editable
+    redirect_to admin_edition_path(@edition) unless @edition.editable?
+  end
```

Comment:
Presumably we could do away with the other 'GDS Admin only' change note controller by tweaking this, right?

(Of course that's not in scope for this spike, but just thinking about how we could avoid maintaining two very similar controllers in future.)

Reply:
Yea i think so. It would require a decent refactor though as the existing controller is built by taking 2 id's, the current edition's and a historical one (for the edition where the change note will be updated). 

It's not immediately obvious to me why the current editions id needs to be passed. I suspect i can probs be removed.

But yes, we could enforce different permissions for editions in different states & only allow GDS admins to see the index endpoint etc.